.github/CONTRIBUTING.md

```diff
@@ -1,13 +1,22 @@
 # Contributing
+![](https://raw.githubusercontent.com/checkstyle/resources/master/img/checkstyle-logos/checkstyle-logo-260x99.png)
 
-Please see instructions for contributing here:
-https://checkstyle.org/contributing.html
+#### [Reporting issues](https://checkstyle.org/report_issue.html)
+#### [Pull Request Template](https://github.com/checkstyle/checkstyle/blob/master/.github/PULL_REQUEST_TEMPLATE.md)
+#### [Issue Template](https://github.com/checkstyle/checkstyle/blob/master/.github/ISSUE_TEMPLATE.md)
+#### [How to report a bug?](https://checkstyle.org/report_issue.html#How_to_report_a_bug)
 
-# Reporting issues
+Please provide issue report in format that we request, EACH DETAIL MAKE A HUGE HELP.
```

Comment:
`in format` => `in the format`.
`DETAIL MAKE A HUGE` => `DETAIL IS A HUGE`

Reply:
Done